### ISSUE TITLE
tags value issues

### ISSUE THREAD

**Issue by emisjerry** [joshua-martius/tagmany#6]
Hi,

tags: tool will changed to t, o, l after invoking tagMany command.

One tag "tool" becomes "t", "o", "l" 3 tags after inserting a new tag.

**Comment by joshua-martius** [comment]
Hello,

thank you for reporting a problem. I just released a new version. Please update and tell me if it works for you now.

Please remember to seperate the tags with commas (e.g. tool1,tool2,tool3).

**Comment by emisjerry** [comment]
Hi, thanks for your reply. But it still cannot work.

My content:
```
---
aliases: 
created: 2023-08-31 17:53:41
modified: 2023-08-31 17:58:06
number headings: first-level 1, max 6, _.1.1.
tags: tool
---
# test4
```

After append new tag:
![image](https://github.com/joshua-martius/tagmany/assets/131511/617b8e7d-d954-4ecc-8b20-4c8965b1eb62)

Is YAML's problem? tags should be a list, but i have used it as string.

**Comment by joshua-martius** [comment]
Okay, now I was able to reproduce it.

For now, I cannot come up with an easy solution so I might have to ask you to use tags in one of the ways I have tested them, being:

```yaml
tags: [tag1,tag2,...]
```

```yaml
 tags:
- tag1
- tag2
- ...
```

The problem seems to be that the function https://github.com/joshua-martius/tagmany/blob/13a23573f8c3ccab78ae6e15e5cf31f8df17b736/src/main.ts#L39 here is processing the tags within the frontmatter as an array. This is within the built-in function and not possible for me to change.
